### DISCUSSION TITLE
Track kWh per RFID card

### DISCUSSION THREAD

**Discussion by nathanielakkermans** [serkri/SmartEVSE-3#253] [Ideas]
For me it would be a nice function if I could keep track of how many kWh are being charged per FRID card. I have a private electric car and a business one, so I would like to split how much power goes to which one. is this something interesting for the project to implement?

**Comment by dingo35** [comment]
That is already a feature requested:

https://github.com/dingo35/SmartEVSE-3.5/issues/31
